File: python_programming_basics/conditional_statements_advanced/l_12_trade_commissions.py
town = input()
sales = float(input())
result = 0
wrong_data = False

if town == "Sofia":
    if 0 <= sales <= 500:
        result = sales * 0.05

    elif 500 < sales <= 1000:
        result = sales * 0.07

    elif 1000 < sales <= 10000:
        result = sales * 0.08

    elif sales > 10000:
        result = sales * 0.12

    else:
        wrong_data = True

elif town == "Varna":
    if 0 <= sales <= 500:
        result = sales * 0.045

    elif 500 < sales <= 1000:
        result = sales * 0.075

    elif 1000 < sales <= 10000:
        result = sales * 0.10

    elif sales > 10000:
        result = sales * 0.13

    else:
        wrong_data = True

elif town == "Plovdiv":
    if 0 <= sales <= 500:
        result = sales * 0.055

    elif 500 < sales <= 1000:
        result = sales * 0.08

    elif 1000 < sales <= 10000:
        result = sales * 0.12

    elif sales > 10000:
        result = sales * 0.145

    else:
        wrong_data = True

else:
    wrong_data = True

if wrong_data:
    print("error")

else:
    print(f"{result:.2f}")

File: JavaScript Basics/03. Conditional Statements Advanced/l-12-Trade-Commissions.js
function calculateCommission(city, sales) {
    if (sales < 0 || !["Sofia", "Varna", "Plovdiv"].includes(city)) {
        console.log("error");
        return;
    }

    let commissionRate;


    if (city === "Sofia") {
        if (sales <= 500) commissionRate = 0.05;
        else if (sales <= 1000) commissionRate = 0.07;
        else if (sales <= 10000) commissionRate = 0.08;
        else commissionRate = 0.12;
    
    } else if (city === "Varna") {
        if (sales <= 500) commissionRate = 0.045;
        else if (sales <= 1000) commissionRate = 0.075;
        else if (sales <= 10000) commissionRate = 0.10;
        else commissionRate = 0.13;
    
    } else if (city === "Plovdiv") {
        if (sales <= 500) commissionRate = 0.055;
        else if (sales <= 1000) commissionRate = 0.08;
        else if (sales <= 10000) commissionRate = 0.12;
        else commissionRate = 0.145;
    }

    let commission = sales * commissionRate;
    console.log(commission.toFixed(2));
}


calculateCommission("Sofia", 1500); 